packages/babel-preset-env/test/fixtures/preset-options/hermes-0_7/options.json

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "hermes": "0.7"
```

Comment:
Where does 0.7 come from? There own stated last version is 0.12.0:

![image](https://github.com/babel/babel/assets/5236548/05b537c1-6ab6-40e5-9c1d-2de56165909e)

Reply:
React native versions are much higher (0.70 and above).

The 0.7 is here because hermes added some new features in 0.7, and this test is verfying wha tis transpiled when targeting that version.

Reply:
I mean the last hermes version is v0.12.0: https://github.com/facebook/hermes/releases
The last react-version is 0.70.14: https://github.com/facebook/react-native/releases

What do you mean by "hermes added some new features in 0.7", since the version was never released? That refers to react-native?

Reply:
Hermes released 0.7.0 in 2020. See: https://github.com/facebook/hermes/releases/tag/v0.7.0

Reply:
@colinaaa Thank you, now it makes sense to me, you were both right... I was highly confused by 0.7 and 0.70 :zany_face: